crates/polars-core/src/chunked_array/temporal/conversion.rs

```diff
@@ -52,6 +52,32 @@ pub(crate) fn naive_datetime_to_date(v: NaiveDateTime) -> i32 {
     (datetime_to_timestamp_ms(v) / (MILLISECONDS * SECONDS_IN_DAY)) as i32
 }
 
-pub(crate) const NS_IN_DAY: i64 = 86_400_000_000_000;
-pub(crate) const US_IN_DAY: i64 = 86_400_000_000;
-pub(crate) const MS_IN_DAY: i64 = 86_400_000;
+pub fn get_strftime_format(fmt: &str, dtype: &DataType) -> String {
+    if fmt != "iso" {
+        return fmt.to_string();
+    }
+    #[allow(unreachable_code)]
+    let fmt: &str = match dtype {
+        #[cfg(feature = "dtype-datetime")]
+        DataType::Datetime(tu, tz) => match (tu, tz.is_some()) {
+            (TimeUnit::Milliseconds, true) => "%F %T%.3f%:z",
+            (TimeUnit::Milliseconds, false) => "%F %T%.3f",
+            (TimeUnit::Microseconds, true) => "%F %T%.6f%:z",
+            (TimeUnit::Microseconds, false) => "%F %T%.6f",
+            (TimeUnit::Nanoseconds, true) => "%F %T%.9f%:z",
+            (TimeUnit::Nanoseconds, false) => "%F %T%.9f",
+        },
```

Comment:
@alexander-beedie I think these formats are not ISO formats.
Doesn't the separator between the date and time have to be `T`?
Is it appropriate to use `iso` as an alias for something that is not an ISO format?

It seems that the true ISO format can be specified with `%+`.
https://docs.rs/chrono/latest/chrono/format/strftime/index.html#fn7

Reply:
It's not quite so straightforward (in this version each _component_ is ISO, with the components joined with a space - and this was considered a valid ISO format itself until a subsequent standard amendment), but... we _should_ have an option that allows for a strict ISO return with "t" as the separator. Think I'll add an "iso:strict" mode to handle this 🤔 (the format with a space is now technically known as RFC 3339, which is a subset of ISO 8601).

Reply:
Thanks for your reply.

> RFC 3339, which is considered a subset of ISO 8601

It does not seem _subset_.
https://ijmacd.github.io/rfc3339-iso8601/

Reply:
The space-separated format is a subset; the entire spec is not.